pinboard_tabs.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'lib/pinboard_tabs_downloader'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.